### ISSUE TITLE
Update High Contrast setup for Checkbox and Radio button

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/Orange-Boosted-Bootstrap#2987]
based on updates from the accessibility team, the high contrast mode should be updated as follows:
* if disabled: nothing specific to do
* if enabled: important semantic components/elements (like checkbox or radio buttons) must use the "color.content.default" color token to be displayed in light & dark mode

Note that this "color.content.default" color can also be the color used for the hover state.
Note also that it is recommended to be able to distinguish the enabled and the hover states.
Therefore when the high contrast mode is activated, the hover state must display the focus border.
